JustSaying/JustSayingFluently.cs

```diff
@@ -187,9 +187,20 @@ public ISubscriberIntoQueue WithSqsPointToPointSubscriber()
 
         #region Implementation of Queue Subscription
 
-        public IFluentSubscription IntoQueue(string queuename)
+        public IFluentSubscription IntoQueueNamed(string queueName)
         {
-            _subscriptionConfig.BaseQueueName = queuename;
+            if (string.IsNullOrWhiteSpace(queueName))
+            {
+                throw new ArgumentNullException(nameof(queueName));

```

Comment:
I mean the whole change. It makes it clearer in 1 case, but makes this case harder:

var queueNameOrNull = GetAQueueNameOrNull();

...

.WithSqsTopicSubscriber()		     
.IntoQueue(queueNameOrNull)
...